lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: kubevirt
+    plugin_type: inventory
+    author:
+      - KubeVirt Team (@kubevirt)
+
+    short_description: KubeVirt inventory source
+
+    description:
+      - Fetch VirtualMachines and VirtualMachinesInstances for one or more clusters
```

Comment:
Description should be: _Fetch running virtual machines from KubeVirt clusters_

Reasoning:
1. This only returns *running* vms.
2. We've decided to hide the internal implementation of VMs/VMIs (as much as possible) from ansible users.